modules/end-user-guide/pages/supported-languages.adoc

```diff
@@ -1,10 +1,18 @@
 :_content-type: REFERENCE
-:description: Supported languages
+:description: Stack samples
 :keywords: user-guide, languages
-:navtitle: Supported languages
+:navtitle: Stack samples
 :page-aliases:
 
 [id="supported-languages"]
-= Supported languages
+= Stack samples
 
 include::example$snip_{project-context}-supported-languages.adoc[leveloffset=+1]
+
+[NOTE]
+====
+The amount of supported languages may vary depending on the selected IDE. For a full list of languages supported by your IDE, refer to its documentation.
```

Comment:
IBM SG wants us to avoid `may` 

I think it reads perfectly fine to simply say, 
`The amount of supported languages vary depending on the selected IDE.`